middlewares/userValidation.js

```diff
@@ -0,0 +1,47 @@
+class UserValidation {
+    static signupValidation(req, res, next) {
+        req.checkBody('email')
+            .notEmpty()
+            .withMessage('Email is required')
+            .trim()
+            .isEmail()
+            .withMessage('Invalid Email address')
+            .customSanitizer(email => email.toLowerCase());
+
+        req.checkBody('firstName')
+            .notEmpty()
+            .withMessage('First name is required')
+            .trim()
+            .isLength({ min: 3, max: 15 })
+            .withMessage('Name should be between 3 to 15 character long')
+            .isAlpha()
+            .withMessage('Name should only contain alphabets');
+
+        req.checkBody('lastName')
+            .notEmpty()
+            .withMessage('last name is required')
+            .trim()
+            .isLength({ min: 3, max: 15 })
+            .withMessage('Name should be between 3 to 15 character long')
+            .isAlpha()
+            .withMessage('Name should only contain alphabets');
+
+        req.checkBody('password')
```

Comment:
Currently, you need to check this in password:
1) Is the password has upper and lowercase characters?
2) Is the password has at least one number?
3) Are password at least 6 letters long?
In order to mimic [front-end validation](https://github.com/zero-to-mastery/mappypals/blob/dev/src/components/helper.js)

Reply:
So I should only work on the password validation

Reply:
You need to improve password validation and remove the minimum limit for name and last name.

Reply:
So do you mean someone can have just a letter 'j' as his first name? Because if you remove the minimum that's what you mean

Reply:
So do you mean someone can have just a letter 'j' as his first name? Because if you remove the minimum that's what you mean

Reply:
Yes according to Mubaraksow most Chinese names, have only 2 letters, so he wants to have no limit. Allow users to have one letter names.

Reply:
ok then make sense, would work on it asap

Reply:
The issue has been resolved, please review